tutorindigo/templates/indigo/lms/static/sass/extra/_header.scss

```diff
@@ -161,6 +170,81 @@ header.global-header {
     }
 }
 
+.theme-toggle-button{
```

Comment:
When `INDIGO_ENABLE_DARK_TOGGLE` is false, add a rule with `display: hidden` to hide the toggle.